# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../darkglass/icondir.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- darkglass-0.0.5/Makefile.orig	2008-04-17 00:54:53.000000000 +0200
+++ darkglass-0.0.5/Makefile	2008-04-17 00:54:59.000000000 +0200
@@ -1,6 +1,6 @@
 
 PREFIX=/usr/X11
-ICONDIR=lib/X11/icons
+ICONDIR=share/icons
 NAME=darkglass
 
 CURSORS=xterm right_ptr basic_arrow hand1 hand2 pirate question_arrow right_side sizing
